internal/awsservice/cloudwatchlogs.go

```diff
@@ -49,20 +39,18 @@ func ValidateLogs(t *testing.T, logGroup, logStream string, numExpectedLogs int,
 
 	numLogsFound := 0
 	attempts := 0
-	var output *cloudwatchlogs.GetLogEventsOutput
 	var nextToken *string
 
 	for {
 		if nextToken != nil {
 			params.NextToken = nextToken
 		}
-		output, err = cwlClient.GetLogEvents(*clientContext, params)
+		output, err := CwlClient.GetLogEvents(cxt, params)
```

Comment:
should `cxt` be renamed `ctx`, just for convention's sake?

Reply:
I'm good with that too ! Thanks for the suggestion.